functions/lib/sql-helper.js

```diff
@@ -173,3 +174,33 @@ module.exports.sqlUpdateAnswersWithResults = async (checkId, answers) => {
   const sql = [insertSql, inserts.join(', \n')].join(' ')
   return sqlService.modify(sql, params)
 }
+
```

Comment:
this 'helper' class is starting to turn into a dumping ground / god class.  we should be creating specific services for things like check calculation.  please move this into a dedicated 'checkWindowScore' service

Reply:
`sql-helper.js` is ONLY for shared functionality - unless the function is used in at least 2 functions please don't put stuff there.

Reply:
@GuyHarwood @jon-shipley was thinking about the same thing, totally agreed.
It is addressed on f8de44c